api/src/controllers/team.js

```diff
@@ -313,6 +316,68 @@ async function updateTeam(req, res, next) {
   }
 }
 
+/**
+ * Clear the team Data from the database on /team/:team_id
+ * @param {} req request object, containing team id in the url
+ * @param {*} res response object, the deleted team document
+ * @returns 200: teamn is deleted
+ * @returns 404: team is not found
+ * @returns 400: team id is not in a valid hexadecimal format
+ */
+async function resetTeamData(req, res, next) {
+  try {
+    const { teamId } = req.params;
+    const { isDeleteFlag } = req.body;
+    await Achievement.deleteMany({ teamId });
+    await Publication.deleteMany({ teamId });
```

Comment:
I think team members from the Team schema needed to be reset to the empty array as well. I still got the previous team members list after I reset data and then adding back the team page.